Rubberduck.VBEEditor/SafeComWrappers/VBA/Windows.cs

```diff
@@ -1,5 +1,8 @@
 using System.Collections;
 using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Microsoft.Office.Interop.Outlook;
```

Comment:
Is that intentional? I don't see a change that warrant uses of the `Microsoft.Office.Interop.Outlook`?

Reply:
No, these changes are not intentional. I will remove them in my next PR.